Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/Asset/AssetManager.h>
+#include <AzCore/Asset/AssetManagerBus.h>
+#include <AzCore/IO/FileIO.h>
+#include <AzCore/Math/Crc.h>
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzCore/std/containers/unordered_set.h>
+#include <AzToolsFramework/API/EditorAssetSystemAPI.h>
+
+namespace ROS2::Utils
+{
+    //! Structure contains essential information about the source and product assets in O3DE.
+    //! It is designed to provide necessary information for other classes in URDF converter, e.g. CollidersMaker or VisualsMaker.
+    struct AvailableAsset
+    {
+        //! Relative path to source asset e.g. `Assets/foo_robot/meshes/bar_link.dae`.
```

Comment:
doxygen will cut off items after the period. 
```suggestion
        //! Relative path to source asset eg `Assets/foo_robot/meshes/bar_link.dae`.
```

Reply:
I see this exact example in Gotchas, nice catch!
Perhaps we should change it to "for example" so that it is grammatically correct?